pages/trips/[id]/index.js

```diff
@@ -139,3 +164,51 @@ export default function DetailsPage() {
     </main>
   );
 }
+
+function PackingListEntry({
+  _id,
+  name,
+  handleDeleteFromPackingList,
+  handleEditFromPackingList,
+}) {
+  const [isEditing, setEditing] = useState(false);
+
+  function onSubmit(event) {
+    event.preventDefault();
+    handleEditFromPackingList(_id, event.target.name.value);
+    setEditing(false);
+  }
+
+  return (
+    <section>
+      {isEditing ? (
+        <form onSubmit={onSubmit}>
+          <label>
+            
+            <input
+              name="name"
+              placeholder="Edit your item"
+              defaultValue={name}
+              required
+              autoFocus
+            />
+          </label>
+          <button>&#10003;</button>
+          <button type="button" onClick={() => setEditing(false)}>
+          &#10680;
+          </button>
+        </form>
+      ) : (
+        <>
+          <span>
+            {name}
+            <button onClick={() => setEditing(true)}>&#9998;</button>
+            <button onClick={() => handleDeleteFromPackingList(_id)}>
+              &#10060;
+            </button>
+          </span>
```

Comment:
It would be better to add aria labels to your buttons, since just using these characters is not accessible. This is purely optional ;)